### PR TITLE
10460: Handle Refactor Feedback

### PR DESCRIPTION
**Pull request by cruzjone-flexion**

### CODE REVIEW

File: scripts/reports/stale-cases.ts [hunk 1]
```diff
@@ -0,0 +1,185 @@
+// usage:
```
Comment (Mwindo) [inline scripts/reports/stale-cases.ts]: Irrelevant to the PR, just missing from test but in staging.

File: web-client/src/ustc-ui/Select/Selects.tsx [hunk 2]
```diff
@@ -1,6 +1,38 @@
...
+  return !!obj.label && !!obj.options;
+}
+
+export function Select({
+  defaultValue,
+  name,
+  onChange,
+  options,
+  value,
+}: SelectParamsType) {
```
Comment (Mwindo) [inline web-client/src/ustc-ui/Select/Selects.tsx:35]: Thank you!

File: web-client/src/views/Public/TrialSessions/PublicTrialSessions.tsx [hunk 3]
```diff
@@ -190,13 +211,25 @@ function MobilePublicTrialsSessions({
 
         {isOpen && (
           <>
-            <PublicTrialSessionsFilters ROOT={ROOT} />
+            <PublicTrialSessionsFilters
```
Comment (Mwindo) [inline web-client/src/views/Public/TrialSessions/PublicTrialSessions.tsx:214]: Beautiful!

**Review by Mwindo — APPROVED** [review]
LGTM (although I shouldn't speak for my changes!). Thanks @cruzjone-flexion and @nechama-krigsman!

File: web-client/src/ustc-ui/Select/Selects.tsx (hunk 2 above)
Comment (nechama-krigsman) [reply web-client/src/ustc-ui/Select/Selects.tsx:35]: Nice!

**Review by nechama-krigsman — APPROVED** [review]

**Pull request merged**